style.css

```diff
@@ -1,3 +1,18 @@
+@media (prefers-color-scheme: dark) {
+    body {
+        background: #1e1e1e;
+        color: #cfcfcf;
+    } 
+    input, thead {
+        background: #e9e9edde !important;          
```

Comment:
What's the reason for using a (partially) transparent background color here, and why `!important`? I'd want to avoid both unless there's a strong reason in favor.